src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -24,6 +25,7 @@ export const JobLogsContainer = props => {
       selectRecord,
       selectedRecords,
     },
+    stripes,
```

Comment:
You can decorate the component into `withStripes` instead of passing it to props